src/components/transfer/Transfer.vue

```diff
@@ -229,22 +230,33 @@ export default {
         }
       },
     },
+    switchNetwork(value) {
+      if (value) {
+        this.resetSelectedToken();
```

Comment:
most of the files in the ui project don't use semi colon (;). We should keep consistency about including semi-colon at the end of the line or not.